app/views/schools/new.html.erb

```diff
@@ -4,15 +4,15 @@
 <%= form_with url: '/schools', method: :post, local: true do |form|%>
   <h3><%= form.label "Create New School:" %></h3>
   <p><%= form.label "Name: " %></p>
-  <p><%= form.text_field :name, placeholder: "Enter Name" %></p><br>
+  <p><%= form.text_field :name, placeholder: "Turing" %></p><br>
   <p><%= form.label "Address: " %></p>
-  <p><%= form.text_field :address, placeholder: "Enter Address" %></p><br>
+  <p><%= form.text_field :address, placeholder: "123 Street" %></p><br>
   <p><%= form.label "City: " %></p>
-  <p><%= form.text_field :city, placeholder: "Enter City" %></p><br>
+  <p><%= form.text_field :city, placeholder: "Denver" %></p><br>
   <p><%= form.label "State: " %></p>
-  <p><%= form.text_field :state, placeholder: "Enter State" %></p><br>
+  <p><%= form.text_field :state, placeholder: "CO", maxlength: 2 %></p><br>
   <p><%= form.label "Zipcode: " %></p>
-  <p><%= form.text_field :zipcode, placeholder: "Enter Zipcode" %></p><br>
+  <p><%= form.text_field :zipcode, placeholder: "80202", maxlength: 5 %></p><br>
```

Comment:
Nice didn't know about this. So this limits the amount of numbers a user can input?

Reply:
yes!